src/components/attendee/AttendeeHeader/AttendeeHeader.tsx

```diff
@@ -70,30 +73,51 @@ export const AttendeeHeader: React.FC<AttendeeHeaderProps> = ({
 
   const { userWithId } = useUser();
 
+  const isNarrow = isTablet || isMobile;
+
   const renderedCaptions = useMemo(
     () =>
-      Object.entries(tabCaptions).map(([key, label]) => (
-        <Button onClick={() => handleOverlayOpen(key)} key={key}>
-          {label}
+      !isNarrow ? (
+        Object.entries(tabCaptions).map(([key, label]) => (
+          <Button onClick={() => handleOverlayOpen(key)} key={key}>
+            {label}
+          </Button>
+        ))
+      ) : (
+        <Button variant="primaryAlternate" onClick={show}>
```

Comment:
I've noted in a comment below, `<Button>` can be made to accept more props than just `variant` for the style.

Reply:
yep, thanks [a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)